src/MSALWrapper/AuthFlow/AuthFlowExecutor.cs

```diff
@@ -33,9 +33,9 @@ public AuthFlowExecutor(ILogger logger, IEnumerable<IAuthFlow> authFlows)
         /// Get a auth flow result.
         /// </summary>
         /// <returns>The <see cref="Task"/>.</returns>
-        public async Task<AuthFlowResult> GetTokenAsync()
+        public async Task<List<AuthFlowResult>> GetTokenAsync()

```

Comment:
I think we want to make it obvious which AuthFlow Result is the successful one, while still returning the list. Only returning a List means our code up top has to now find the successful , or no successful attempt.

Reply:
Discussed offline to go with this approach.